Cargo.toml

```diff
@@ -18,6 +18,8 @@ chrono = "0.4.23"
 tantivy = "0.20.1"
 itertools = "0.10.5"
 futures = "0.3.26"
+pythonize = "0.19.0"
+serde = "~1.0"
```

Comment:
Why a non-default version specification for the requirement on `serde`?

Reply:
Sorry, I was unspecific. Why do you use a tilde requirement `~1.0` meaning `>=1.0, <1.1` instead of the default type `1.0` meaning `^1.0` meaning `>=1.0, <2.0`?

Reply:
I wanted to be more conservative/more explicit with the permissible versions, since unexpected changes in `serde` could make previously pickled types might not be compatible. But `1.0` (aka `^1.0`) should be fine, since breaking changes aren't likely with minor version changes.